xml/admin_docupdates.xml

```diff
@@ -26,6 +26,19 @@
   <sect2 xml:id="sec.admin.docupdates.sle15.sp0.nov_2018">
    <title>November 2018</title>
    <variablelist>
+    <varlistentry>
+     <term>New sections</term>
```

Comment:
@lproven : In order to make it similar to the rest of the docupdate entries, could you insert an xref to the chapter ID please (=using the chapter title as a term)? Then you could remove the xref from the para.

Reply:
Done.